clio-client/src/test/scala/org/broadinstitute/clio/client/webclient/GoogleCredentialsGeneratorSpec.scala

```diff
@@ -15,50 +15,46 @@ class GoogleCredentialsGeneratorSpec extends FlatSpec with Matchers {
   behavior of "GoogleTokenGenerator"
 
   it should "refresh uninitialized OAuth2Credentials" in {
-    val mockCreds = new MockOAuth2Credentials(None)
+    val mockCreds = new MockOAuth2Credentials()
     val generator = new GoogleCredentialsGenerator(mockCreds)
 
-    mockCreds.getAccessToken should be(null)
     mockCreds.getAccessToken should be(null)
     val token = generator.generateCredentials()
     mockCreds.getAccessToken shouldNot be(null)
     mockCreds.getAccessToken.getTokenValue should be(token.token)
   }
 
   it should "refresh expired OAuth2Credentials" in {
-    val oldTokenString = "abcdefg"
-
-    val oldToken =
-      new AccessToken(oldTokenString, Date.from(Instant.now().minusSeconds(10)))
-    val mockCreds = new MockOAuth2Credentials(Some(oldToken))
+    // Negative duration so each refresh produces an already-expired token.
+    val mockCreds = new MockOAuth2Credentials(-1)
     val generator = new GoogleCredentialsGenerator(mockCreds)
 
-    generator.generateCredentials().token shouldNot be(oldTokenString)
-    mockCreds.getAccessToken shouldNot be(oldToken)
+    val token1 = generator.generateCredentials()
+    val token2 = generator.generateCredentials()
+
+    token1.token shouldNot be(token2.token)
+    mockCreds.getAccessToken.getTokenValue should be(token2.token)
   }
 
   it should "not refresh unexpired OAuth2Credentials" in {
-    val oldTokenString = "abcdefg"
-
-    val oldToken =
-      new AccessToken(oldTokenString, Date.from(Instant.now().plusSeconds(10)))
-    val mockCreds = new MockOAuth2Credentials(Some(oldToken))
+    val mockCreds = new MockOAuth2Credentials(100)
     val generator = new GoogleCredentialsGenerator(mockCreds)
 
-    generator.generateCredentials().token should be(oldTokenString)
-    mockCreds.getAccessToken should be(oldToken)
+    val token1 = generator.generateCredentials()
+    val token2 = generator.generateCredentials()
+
+    token1 should be(token2)
   }
 }
 
 object GoogleCredentialsGeneratorSpec {
 
-  class MockOAuth2Credentials(initToken: Option[AccessToken])
-      extends OAuth2Credentials(initToken.orNull) {
```

Comment:
The one-arg constructor for `OAuth2Credentials` is now deprecated (and we've turned all warnings into errors through our compiler flags), so I had to shift this test around.